web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1663,116 +1663,178 @@ export const formatters: Formatters = {
   [eventCodes.ACCESS_LIST_CREATE]: {
     type: 'access_list.create',
     desc: 'Access list created',
-    format: ({ name, updated_by }) =>
-      `User [${updated_by}] created access list [${name}]`,
+    format: ({ access_list_title, name, updated_by }) => {
+      return (
+        `User [${updated_by}] created access list ` +
+        (access_list_title ? `[${access_list_title}]` : `[${name}]`)
```

Comment:
nit: inline them instead of individual vars and `+`
```suggestion
         `User [${updated_by}] created access list [${access_list_title ||  name}]`
```

Applied to all the events below.